module_evaluations/TCR-32_2023-09-06.MD

```diff
@@ -0,0 +1,79 @@
+  * missing `mod-fqm-manager` (NB: this interface appears to be misnamed; it should not have the `mod-` prefix)
+* [x] Third party dependencies use an Apache 2.0 compatible license
+  * several dependencies licenses are undefined, though not strictly speaking incompatible
+* [x] ~Installation documentation is included~
+* [x] Personal data form is completed, accurate, and provided as `PERSONAL_DATA_DISCLOSURE.md` file
+  * It isn't clear to me that personal data is actually stored/cached in cookies, but they are being transparent about the kinds of data this module handles.
+* [x] Sensitive and environment-specific information is not checked into git repository
+* [x] Module is written in a language and framework from the [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page
+  * In fact there is no list of officially supported front-end languages. This module is written in TypeScript, which is uncommon among front-end modules, but which is nonetheless specifically supported by the UI build tools.
+* [ ] Module only uses FOLIO interfaces already provided by previously accepted modules _e.g. a UI module cannot be accepted that relies on an interface only provided by a back end module that hasn't been accepted yet_
+  * TCRs are open for all three modules leveraged by this one.
+* [ ] Module gracefully handles the absence of third party systems or related configuration
+  * I was unable to successfully run the module locally and therefore cannot evaluate this
+* [x] Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication
+* [x] Uses [officially supported](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) build tools
+* [x] Unit tests have 80% coverage or greater, and are based on [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)
+  * APIs are mocked via miragejs, which is not the most common way to handle this task.
+
+## Frontend
+* [x] If provided, End-to-end tests must be written in an [officially approved technology](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)
+  * not provided
+* [x] Have i18n support via react-intl and an `en.json` file with English texts
+* [ ] Have WCAG 2.1 AA compliance as measured by a current major version of axe DevTools Chrome Extension
+  * I was unable to successfully run the module locally and therefore cannot evaluate this
+* [x] Use the latest release of Stripes at the time of evaluation
+  * Evaluated against stripes v9 (Poppy), which has not been formally released. Evaluating against v8 (Orchid) would have no value since this would effectively be evaluating a new module for inclusion in a Service Patch for a past release instead of a future one.
+* [ ] Follow relevant existing UI layouts, patterns and norms
+  * I was unable to successfully run the module locally and therefore cannot evaluate this
```

Comment:
In general this seems to be true.  On the "New" page, the left "Set criteria" sidebar is new to me.  Is this an existing pattern?